### ISSUE TITLE
[Question] How to always place `[No Name]` buffers at the right hand side from the list?

### ISSUE THREAD

**Issue by nyngwang** [akinsho/bufferline.nvim#383]
As title. While I want to keep the newly opened buffer have the behavior on the current `dev` branch.

**Comment by akinsho** [comment]
@nyngwang you can create a `group` that matches any buffer that has no name and place those last in the grouping order.

```lua
local groups = require('bufferline.groups')
config = {
	options = {
		groups = {
			items = {
	          	groups.builtin.ungrouped,
				{matcher = function(b) return b.name == "" end, name = "unnamed"}
			}
		}
	}
}
```

Otherwise, you can create a custom sorter but will lose the newly opened on the right behaviour 🤷🏾‍♂️.

Please experiment in your config having read through the various available options, you can probably figure out the right solution for yourself.

**Comment by nyngwang** [comment]
> Otherwise, you can create a custom sorter but will lose the newly opened on the right behaviour 🤷🏾‍♂️.

Yes, I was also thinking about asking you for this. I was trying to extend those sorters you have written. But it seems that the `state` will not be passed into the custom function I provided via `options.sort_by`. Anyway, let me try it now.

**Comment by nyngwang** [comment]
@akinsho Is it possible to re-run the sorting again after I re-order one of these buffers?

**Comment by akinsho** [comment]
You can use the sort by function documented in the README with the string for the type of sort to run but I don't know if that will preserve the no name positioning 🤷🏾‍♂️
